bin/templates/project/legacy/build.gradle

```diff
@@ -223,22 +223,6 @@ android {
             }
         }
     }
-    /*
-
-    ELSE NOTHING! DON'T MESS WITH THE VERSION CODE IF YOU DON'T HAVE TO!
-
-    else if (!cdvVersionCode) {
-      def minSdkVersion = cdvMinSdkVersion ?: privateHelpers.extractIntFromManifest("minSdkVersion")
-      // Vary versionCode by the two most common API levels:
-      // 14 is ICS, which is the lowest API level for many apps.
-      // 20 is Lollipop, which is the lowest API level for the updatable system webview.
-      if (minSdkVersion >= 20) {
-        defaultConfig.versionCode += 9
```

Comment:
@dpogue Can you confirm if this is safe to remove?

I feel like this may affect the multiple apk build. That part of the code also reads from the `defaultConfig.versionCode`

I am thinking this should be at least

```
else if (!cdvVersionCode) {
  defaultConfig.versionCode += 9
}
```

Reply:
It's commented out code, so it's not doing anything at the moment.

Reply:
Haha, I wonder how that slipped past me, why I didn't notice it was commented out. 🤣